### DISCUSSION TITLE
First Project

### DISCUSSION THREAD

**Discussion by Koocho** [TinkerTools/tinker#111] [Q&A] [answered]
Hi all,
I hope this is an appropriate channel to ask questions.
I am trying to create a waterbox of TIP3P water with an Amber99 force field, I am a complete beginner, so any pointers would be greatly appreciated.

Thank you in advance!

**Comment by jacekkozuch** [comment, marked as answer]
Hi,

Based on the coordinates, just create a water.xyz file (manually or using some program according to the tinker format), i.e.
3
1 O x-coord y-coord z-coord type 2 3
2 H ...           ...            ...           type 1
3 H ...           ...             ...           type 1

(Types need to be according to amber force field file in Tinker - I suppose, I only worked with amoeba in Tinker. I assume the amoeba and amber formatting is the same).

Then you need a water.key with the line
parameters path_to_amberprm/amber99.prm
(Further lines will be needed depending on what you want to do later)

Then you can use the "xyzedit" function to "create and fill box" (option 23, if I remember correctly).
On TinkerTools there are a few tutorials that I would recommend you to try out and get the idea of how Tinker works.

Best,
Jacek
